### PR TITLE
Define a way to define fixture objects and traverse them

### PR DESCRIPTION
**Pull request by bechang**
Add a way to specify our test fixtures in a safe(r) way.

### CODE REVIEW

File: src/test/scala/edu/colorado/plv/cuanto/testing/implicits.scala [hunk 1]
```diff
@@ -4,13 +4,46 @@ import org.scalactic.Equality
 
 import scala.util.Try
 
-/**
+/** Defines convenience conversions that should be solely used in test code.
```
Comment (bennostein) [inline src/test/scala/edu/colorado/plv/cuanto/testing/implicits.scala:7]: Is there a way to enforce this?

File: src/test/scala/edu/colorado/plv/cuanto/testing/implicits.scala [hunk 2]
```diff
@@ -4,13 +4,46 @@ import org.scalactic.Equality
...
+    private lazy val omirror = scala.reflect.runtime.currentMirror.reflect(f)
+
+    /** Makes the `lazy val`s of type `T` as a traversable sequence (via run-time reflection). */
+    def asTraversable[T](implicit ftag: ru.TypeTag[F], ttag: ru.TypeTag[T]): Traversable[(ru.Symbol,T)] = {
+      val typeF = ru.typeOf[F]
+      val typeT = ru.typeOf[T]
+      typeF.decls filter { d =>
+        val tryIsTypeT = for {
+          d <- Try(d.asTerm)
+          dtype = d.infoIn(typeF)
```
Comment (bennostein) [inline src/test/scala/edu/colorado/plv/cuanto/testing/implicits.scala:39]: is this `lval = rval` syntax equivalent to the `lval <- rval` syntax that seems more common?

**Review by bennostein — APPROVED** [review]
Looks good to me, modulo the two minor questions asked inline.

File: src/test/scala/edu/colorado/plv/cuanto/testing/implicits.scala (hunk 1 above)
Comment (bechang) [reply src/test/scala/edu/colorado/plv/cuanto/testing/implicits.scala:7]: It already is by being under src/test.

File: src/test/scala/edu/colorado/plv/cuanto/testing/implicits.scala (hunk 2 above)
Comment (bechang) [reply src/test/scala/edu/colorado/plv/cuanto/testing/implicits.scala:39]: So = is normal let-finding (versus <- is monadic bind).

**Pull request merged**